package.json

```diff
@@ -27,5 +27,11 @@
     },
     "devDependencies": {
         "@symfony/webpack-encore": "^0.31.0"
+    },
+    "scripts": {
+        "dev-server": "encore dev-server",
+        "dev": "encore dev",
+        "watch": "encore dev --watch",
+        "build": "encore production --progress"
```

Comment:
Since this is used in the deploy, how does the progress UI interact with saving the deploy output to a file (for attaching to the release ticket when deploying to stable and prod)?

Reply:
Resolved in 287383a